pinpoint-collector/build/config/pinpoint-collector.properties

```diff
@@ -5,4 +5,4 @@
 # ex) PINPOINT_ZOOKEEPER_ADDRESS should be commented in docker-compose to use pinpoint.zookeeper.address below
 ##########
 #
-# pinpoint.zookeeper.address=
\ No newline at end of file
+pinpoint.zookeeper.address=zoo1
```

Comment:
this change is not needed since this property file is for overriding configurations as explained with the comment in the file

Reply:
Oh, Okay I'll delete it.